sympy/geometry/parabola.py

```diff
@@ -406,3 +407,30 @@ def vertex(self):
             vertex = Point(focus.args[0], focus.args[1] - self.p_parameter)
 
         return vertex
+def tangent():
+     x,y=symbols('x y')
+    print('Enter the parameters a,b,g,f,h,c of equation ax^2+by^2+2gx+2fy+2hxy+c=0')
+    a=int(input())
+    b=int(input())
+    g=int(input())
```

Comment:
We don't want to create interactive functions like this in general

Reply:
I just wanted to show that we can find the equation using this method also.Further this function can be rebuild accordingly